py/client-ticking/src/pydeephaven_ticking/_core.pyx

```diff
@@ -247,63 +247,59 @@ cdef class ColumnSource:
         cdef bool[::1] null_flags_view = null_flags
         boolean_chunk = CGenericChunk[bool].CreateView(&null_flags_view[0], size)
         cdef CGenericChunk[bool] *null_flags_ptr = &boolean_chunk
-        arrow_type: pa.DataType
 
-        element_type_id = CCythonSupport.GetElementTypeId(deref(self.column_source))
+        element_type = deref(self.column_source).GetElementType()
+
+        if element_type.ListDepth() > 1:
```

Comment:
Is this a Python only limitation or does it also extend to C++ clients?  This will need to be part of our updated documentation to reflect what's supported/not supported.

Can we include the element name or type instead of just the depth in the error message?

Reply:
It extends to all the clients as well as to the server. The server's schema representation isn't expressive enough to represent deeply nested types. The reason for this exception is to serve as a reminder in case the situation changes on the server.

Message changed to include element type.